package.json

```diff
@@ -81,12 +81,20 @@
                         "internalDebuggerPort": 9090,
                         "sourceMaps": true,
                         "outDir": "${workspaceRoot}/.vscode/.react"
+                    },
+                    {
+                        "name": "Attach to packager",
+                        "program": "${workspaceRoot}/.vscode/launchReactNative.js",
+                        "type": "reactnative",
+                        "request": "launch",
+                        "internalDebuggerPort": 9090,
+                        "sourceMaps": true,
+                        "outDir": "${workspaceRoot}/.vscode/.react"
                     }
                 ],
                 "configurationAttributes": {
                     "launch": {
                         "required": [
-                            "platform",
```

Comment:
If we go the route of not having a special "attach" platform, I think I'd prefer to change the request type to "attach" and make the changes necessary for that rather than just omitting the platform which could surprise/confuse users.